src-built-in/components/processMonitor/app.jsx

```diff
@@ -5,6 +5,6 @@
 import React from "react";
 import ReactDOM from "react-dom";
 
-import { ProcessMonitor } from "@chartiq/finsemble-ui/src/components/processMonitor/processMonitor";
+import { ProcessMonitor } from "@chartiq/finsemble-ui/react/components/processMonitor/processMonitor";
```

Comment:
Same deal as above. This should be imported from components index, not the actual file.